<?xml version="1.0" ?>
<launch>
  <arg name="robot_ip" default="192.168.131.40" />
  <arg name="load_gripper" default="true" />
  <include file="$(find franka_control)/launch/franka_control.launch" >
    <arg name="robot_ip" value="$(arg robot_ip)" />
    <arg name="load_gripper" value="$(arg load_gripper)" />
  </include>

  <rosparam command="load" file="$(find franka_example_controllers)/config/franka_example_controllers.yaml" />
  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen"  args="cartesian_pose_example_controller"/>
  <!--node pkg="rviz" type="rviz" output="screen" name="rviz" args="-d $(find franka_example_controllers)/launch/robot.rviz"/-->
  <!--node pkg="rostopic" type="rostopic" name="rostopic" args="pub {pub args}"/-->

  <node pkg="joy" type="joy_node" name="logitech_joy" >
    <param name="deadzone" value="0.1" />
    <param name="autorepeat_rate" value="30" />
  </node>

  <node pkg="panda_arm" type="remap_joy_to_point.py" name="franka_joystick_pose_mapper" output="screen" required="true"/>

</launch>

